xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-ui/src/main/resources/CKEditor/Translations.xml

```diff
@@ -266,6 +266,11 @@ ckeditor.plugin.focusedplaceholder.placeholder.caption=Title
 ckeditor.plugin.focusedplaceholder.placeholder.th=Header
 ckeditor.plugin.focusedplaceholder.placeholder.td=Cell
 
+ckeditor.plugin.realtime.notification.sourcearea.temporarilyLeftSession=Temporarily leaving the realtime session.
+ckeditor.plugin.realtime.notification.sourcearea.rejoiningSession.noChanges=No changes were made. Re-joining the realtime session.
+ckeditor.plugin.realtime.notification.sourcearea.rejoiningSession.alone=There are unsaved changes, but no one else is in the realtime session. Re-joining.
+ckeditor.plugin.realtime.notification.sourcearea.notRejoiningSession=There are unsaved changes. Permanently leaving the realtime session.
```

Comment:
I feel "permanently" is too strong. The user can still rejoin if they save & view or cancel and then edit again (or reload the edit mode). Maybe it's enough to say that "We couldn't rejoin the realtime session because there are unsaved changes".